### ISSUE TITLE
F1211 09:46:40.930719  4989 image_io.cpp:135] Check failed: end_frm <= num_of_frames (128 vs. 127) end frame must less or equal to num of frames

### ISSUE THREAD

**Issue by aisunshinehui** [facebook/C3D#338]
when I use C3Dv1.0/c3d_sport1m_feature_extraction_video.sh to extract feature,get the error as the follow:
1211 09:46:35.232805  4770 net.cpp:127] Top shape: 50 487 1 1 1 (24350)
I1211 09:46:35.232811  4770 net.cpp:154] prob needs backward computation.
I1211 09:46:35.232817  4770 net.cpp:76] Creating Layer accuracy
I1211 09:46:35.232823  4770 net.cpp:86] accuracy <- prob
I1211 09:46:35.232828  4770 net.cpp:86] accuracy <- label
I1211 09:46:35.232836  4770 net.cpp:112] accuracy -> accuracy
I1211 09:46:35.232842  4770 net.cpp:127] Top shape: 1 2 1 1 1 (2)
I1211 09:46:35.232847  4770 net.cpp:154] accuracy needs backward computation.
I1211 09:46:35.232852  4770 net.cpp:165] This network produces output accuracy
I1211 09:46:35.232869  4770 net.cpp:183] Collecting Learning Rate and Weight Decay.
I1211 09:46:35.232883  4770 net.cpp:176] Network initialization done.
I1211 09:46:35.232925  4770 net.cpp:177] Memory required for Data 5653289408
E1211 09:46:39.698056  4770 extract_image_features.cpp:81] Extracting features for 4970 batches
F1211 09:46:40.930719  4989 image_io.cpp:135] Check failed: end_frm <= num_of_frames (128 vs. 127) end frame must less or equal to num of frames
*** Check failure stack trace: ***
Aborted (core dumped)
and my input file list like this :+1: 
   input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 0 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 16 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 32 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 48 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 64 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 80 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 96 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 112 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 128 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01.avi 144 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02.avi 0 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02.avi 16 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02.avi 32 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02.avi 48 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02.avi 64 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02.avi 80 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02.avi 96 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 0 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 16 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 32 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 48 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 64 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 80 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 96 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 112 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 128 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 144 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 160 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 176 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 192 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 208 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 224 0
input/avi/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03.avi 240 0
and my output prefix like this:
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000000
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000016
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000032
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000048
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000064
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000080
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000096
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000112
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000128
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01/000144
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/000000
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/000016
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/000032
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/000048
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/000064
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/000080
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02/000096
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000000
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000016
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000032
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000048
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000064
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000080
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000096
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000112
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000128
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000144
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000160
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000176
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000192
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000208
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000224
output/c3d/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c03/000240
please tell me where wrong in my file? Thanks!

**Comment by JerryYaoGl** [comment]
the number ( in each line ) + 16 should be < length of video.

**Comment by dutran** [comment]
What @JerryYaoGl suggested is right, please check your boundary cases @aisunshinehui

**Comment by kjsr7** [comment]
> What @JerryYaoGl suggested is right, please check your boundary cases @aisunshinehui

Hi, Dutran,
Could you tell me how do I calculate the upper limit of the number in each line for a new video other than that of the example videos?
